README.md

```diff
@@ -1,10 +1,37 @@
 <a href="https://github.com/CityOfLosAngeles/angeleno-my-account-flutter/actions"><img src="https://github.com/CityOfLosAngeles/angeleno-my-account-flutter/workflows/Dart%20Analyzer/badge.svg" alt="Dart Analyzer Status"></a>
 <a href="https://github.com/CityOfLosAngeles/angeleno-my-account-flutter/actions"><img src="https://github.com/CityOfLosAngeles/angeleno-my-account-flutter/workflows/Flutter%20Unit%20Tests/badge.svg" alt="Flutter Tests Status"></a>
 [![codecov](https://codecov.io/gh/CityOfLosAngeles/angeleno-my-account/graph/badge.svg?token=ILNR5XOM40)](https://codecov.io/gh/CityOfLosAngeles/angeleno-my-account)
 
+## What is Angeleno Account?
+An Angeleno Account acts as a single sign-on for our public to access multiple City of Los Angeles services that have decided to accept the Angeleno Account credentials. 
+
+## What is Angeleno - My Account?
+Angeleno - My Account is a Flutter based app that interfaces Auth0. This repository is for setting up a Flutter Web Application on your machine to interface Auth0. With Auth0 you can authenticate and authorize user to access your applications. With this repository you will get a Auth0 Web login which then redirects back to the Flutter app in where the authenticated user can see their current profile along with make changes to their profile. We have added Google Places API to assist user's with autocompleting and validating their address.
+
+
 ## Getting Started
-This branch should act as our main branch until we have a working Minimum Viable Product (MVP) to merge into the `main` branch. To start development you'll `git clone` this repo; after cloning you'll by default be on the main branch, but you can `git checkout mvp-skeleton` to switch to the branch with the code being worked on. From the `mvp-skeleton` branch, you can branch off to work on your own work/issue by running `git checkout -b your-branch-name`. When opening the pull request for your work, make sure the branch is being merged into `mvp-skeleton` and not `main`.
+The development branch is our main branch you can use to work on your own work/issues. To start development you'll `git clone` this repo; after cloning you'll by default be on the development branch. When opening the pull request for your work, make sure the branch is being merged into `development` branch as well.
+
+
+### Minimum Requirements
+- Flutter >=3.16.0 [Flutter SDK](https://docs.flutter.dev/get-started/install)
+- Firebase Local Emulator Suite [Emulator Suite](https://firebase.google.com/docs/emulator-suite)
+- Google Cloud Account (https://cloud.google.com/)
+  - Create Project and enable Places API
+    - Places Autocomplete (https://developers.google.com/maps/documentation/places/web-service/autocomplete)
+    - Places Details (https://developers.google.com/maps/documentation/places/web-service/details)
+      - We will consider using Geocoding API for cost optimization (https://developers.google.com/maps/documentation/geocoding/overview#results)
```

Comment:
Additional APIs we need can be found here: https://github.com/CityOfLosAngeles/angeleno-my-account-terraform/pull/18/files

Reply:
serviceusage.googleapis.com",
      "firebase.googleapis.com"
      "firebase.googleapis.com",
      "cloudresourcemanager.googleapis.com",
      "secretmanager.googleapis.com",
      "compute.googleapis.com"


Can you provide the link for the documentation for 
Service Usage
Cloud Resource Manager
Secret Manager API
Compute Engine API


Can you provide the exact Firebase API's, I don't just see one on the console, or can you tell me which ones exactly do you turn on since it seems the line is repeated on PR #18, on the Google Cloud Console? We have to let users know which one to enable on the console since we are not expecting them to use terraform and should be able to do it manually.